common/persistence/class.PhpFileDriver.php

```diff
@@ -260,30 +260,29 @@ private function writeFile($id, $value, $preWriteValueProcessor = null)
 
     /**
      * Create directory and suppress warning message
-     * @param $path
+     * @param string $path
      * @param int $mode
      * @return bool
      */
     private function makeDirectory(string $path, int $mode)
```

Comment:
Could you specify return type according to a usage of that method?

Reply:
added viod